src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -149,7 +151,8 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
       updateVote: (data, dataVote) => updateVoteParentPostHook(data, dataVote, itemParentProps),
       updateVoteLatestChildren: (data, dataVote) =>
         updateVoteLatestChildrenParentHook(data, dataVote, itemParentProps),
-      getComment: getThisComment
+      getComment: getThisComment,
+      getCommentParent: params.getComment
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `getCommentParent: params.getComment` seems to be using the `params` object obtained from `useRoute()`. Ensure that `params.getComment` is always defined and is a function before using it in this context.

```diff
+ getCommentParent: params.getComment
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->